include/cglm/vec4.h

```diff
@@ -256,6 +258,44 @@ glm_vec4_norm(vec4 v) {
 #endif
 }
 
+/*!
+ * @brief L1 norm of vec4
+ *
+ * @param[in] v vector
+ *
+ * @return L1 norm
+ */
+CGLM_INLINE
+float
+glm_vec4_norm_one(vec4 v) {
+#if defined(CGLM_SIMD)
+  return glmm_norm_one(glmm_load(v));
```

Comment:
`CGLM_SIMD` is defined when one of SIMD is supported. If NEON_FP is present then this will (or should) be defined. But it seems `glmm_norm_one()` is only defined for x86. In this case it you should test SIMD against `CGLM_SIMD_x86` not `CGLM_SIMD`, otherwise it cannot be compiled on ARM I guess.

Reply:
Yes, you are absolutely right. I will try to implement `glmm_norm_one()` for ARM and keep the `CGLM_SIMD` definition. Otherwise I will switch to `CGLM_SIMD_x86`, as you said.

Reply:
Can we add more detail to brief of these functions;

Some resources:
https://medium.com/@montjoile/l0-norm-l1-norm-l2-norm-l-infinity-norm-7a7d18a4f40c
https://machinelearningmastery.com/vector-norms-machine-learning/

Also adding new functions to documentation would be awesome if you have enough time 🤗 You can find `docs` folder, cglm uses **sphinx** for documentation.

Reply:
Hi Recep, I will definitely add more info to the brief of the two norm functions. That's no problem. However, I am currently working on two projects at once and I don't have time to update the documentation.

Reply:
> I don't have time to update the documentation.

No problem, I'll update it asap.